src/bindings/js/node/include/validation.hpp

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#include <napi.h>
+
+#include "node/include/addon.hpp"
+#include "openvino/openvino.hpp"
+
+/** @brief Checks if Napi::CallbackInfo value at specified index is an Object.*/
+bool check_object(const Napi::CallbackInfo& info, int index);
```

Comment:
I am not sure that we should pass index here. It would be better to check object with specified type, like:
```suggestion
bool check_object(Napi::Value value); // with this signature `check_object` helper looks unnecessary, use info[index].isObject() where it needs
```